app/form/v1_0/fields/common/utils.ts

```diff
@@ -34,7 +34,12 @@ export function validateFutureDate(value: string) {
 export function validatePastDate(value: string) {
   const now = DateTime.now().startOf('day')
   const date = DateTime.fromISO(value)
-  return !value || value === '' ? true : date.isValid && date <= now
+  return !value || value === '' ? true : date.isValid && date >= now
+}
+
+export function validateValidDate(value: string) {
```

Comment:
There's no 3 functions that do specific things but the code is very similar. Not sure if this is okay or are there better ways on coding this?

Reply:
Good spot! there is definitely common functionality across these three validators which could be abstracted out in to it's own function, which I think sacrifices some simplicity in favour of maintainability, feel free to give refactoring a go